toc/rfc/rfc-draft-pcap-bosh.md

```diff
@@ -2,60 +2,214 @@
 [meta]: #meta
 - Name: Integrate pcap feature into BOSH
 - Start Date: 2023-07-06
-- Author(s): @domdom82 @maxmoehl
+- Author(s): @domdom82 @maxmoehl @peanball
 - Status: Draft
 - RFC Pull Request: https://github.com/cloudfoundry/community/pull/640
 
 
 ## Summary
 
-Members of the [App Runtime Platform WG](https://github.com/cloudfoundry/community/blob/main/toc/working-groups/app-runtime-platform.md) are developing a BOSH release called [pcap-release](https://github.com/cloudfoundry/pcap-release) which adds network traffic capturing tools to both BOSH deployments and Cloud Foundry applications.
-The release was created after an initial discussion was started on the [cf-deployment](https://github.com/cloudfoundry/cf-deployment/issues/980) repository.
+Add a feature to BOSH that provides a convenient network capturing mechanism for instances deployed via BOSH and easy streaming of network traffic to the user issuing the capture request.
 
-Since then, the project has evolved into a BOSH use case and a Cloud Foundry use case. This RFC discusses a proposal to get the BOSH use case integrated with BOSH director, BOSH agent and BOSH cli.
+Capture targets are identified automatically based on metadata in the BOSH Director, i.e. their deployment, instance groups and/or specific instance IDs.
 
-The main goal of the pcap feature is to allow users (BOSH operator in this case) to easily capture network traffic across multiple VMs by specifying various parameters in PCAP CLI:
-- BOSH deployment
-- BOSH instance group(s)
-- BPF [Berkeley Packet Filter](https://www.tcpdump.org/manpages/pcap-filter.7.html) filter to apply
```

Comment:
I wouldn't necessarily remove this section. Even the BOSH lite case needs a list of parameters to identify what to capture. Above you mentioned only "metadata, i.e. deployment, instance groups..." but I would keep the argument list intact. The only difference is that the arguments now go into BOSH CLI instead of PCAP-BOSH CLI.

Reply:
I retained most of this section and added the resulting capture target identification based on these parameters after.